RocketChat/Rocket.Chat
# fix: Manually approved user registration flow

## Proposed changes (including videos or screenshots)



If the Setting `Manually Approve New Users` is enabled and a user tries to create an account, the user gets stuck at the registration form (no redirect happens, but the user is created as Disabled).

**Previous:**
![image](https://github.com/RocketChat/Rocket.Chat/assets/20212776/c7e24840-092d-41a1-896d-43f5a06e14c3)

To improve this behavior the user now gets redirected to the login page with a toast message saying: 

> Before you can login, your account must be manually activated by an administrator.

**NEW:**
P.S: The token it's appearing instead of text due to other issue
![image](https://github.com/RocketChat/Rocket.Chat/assets/20212776/eab08088-cad9-4443-9415-941a392e57d9)



## Issue(s)


behavior identified by this issue: https://github.com/RocketChat/Rocket.Chat/issues/29411

## Steps to test or reproduce


1. As an admin go to your rocket.chat instance 
2. Enable the Setting `Manually Approve New Users` at Account > Registration
3. As a new user, try to register a new account
4. The user is created but no toast message is displayed and the user don't get redirect to other page

## Further comments

WM-70
